lib/src/operance_data_column_width.dart

```diff
@@ -11,7 +11,7 @@ class OperanceDataColumnWidth {
     this.factor = 0.15,
   }) : assert(
           factor > 0 && factor <= 1,
```

Comment:
The assertion could be modified to only check the factor when size is null. This allows for more flexibility in setting the factor when a fixed size is provided.

```suggestion

}) : assert(
          size != null || (factor > 0 && factor <= 1),
          'factor must be between 0 and 1 when size is not provided',
        );


```